src/entries/background/procedures/popup/wallet/accounts.ts

```diff
@@ -0,0 +1,7 @@
+import { getAccounts } from '~/core/keychain';
```

Comment:
I'm wondering if we should bundle contract and procedure implementations to prevent sprawl and better contain "Wallet Actions" under the conceptual umbrella of "Keychain" to logically separate a orpc client/server from the entrypoints in which they are used. MetaMask has a concept of Controllers that could be helpful here. We could have `/core/keychain` as well as `/controllers/keychain` for a `KeychainController` that includes the procedures/schemas. In MetaMask's case, they have a parent controller and then they forward all messages from each entrypoint into the parent and subcontrollers

https://deepwiki.com/MetaMask/metamask-extension/3.2-controller-state
https://deepwiki.com/search/describe-how-messengers-and-co_b467d593-e8cd-4bf4-9a66-013c234b3aa6